doc/Ontology.md

```diff
@@ -0,0 +1,208 @@
+# ACM Knowledge Graph Ontology
+
+## Overview
+
+For details on Knowledge Graphs you can reference other resources.  Perhaps
+start with the [README.md](../README.md) in this repository to begin 
+learning more.
+
+To get started with Knowledge Graphs in ACM, we want our nodes to consist
+of clearly defined entities.  Where possible we want to always re-use the
+existing entities that have been defined instead of creating new ones.
+Validation is being put in place to help prevent too many entities from
+being created.  Too many entity types will make knowledge graph contributions 
+and maintenance more difficult.
+
+## Entities
+
+The list of entities are validated from the [Glossary](../glossary.txt) file
+and consist of the common resources shown below.
+
+![](./ontology.png)
+
+See the examples in the [input](../input) folder for examples on how this
+ontology is being used.
+
+## Entity Descriptions
+
+Entities are the nodes in the knowledge graph.  The following is a list of the
+main entities that are being used in the samples.  While more entities can
+be added, they must not duplicate the ones that already exist.
+
+Entity | Description | Samples | Challenges 
+---    | ---         | ---     | ---
+Users | Users of the ACM APIs. This would also include groups and different User types, such as usage by other Operators or teams. | Customer, Developer, QE, Support | 
```

Comment:
Knowing that we are not distinguishing between different kinds of Users now, should we change the wording a bit?

Reply:
I changed the wording